CHANGELOG.rst

```diff
@@ -26,7 +26,8 @@
 * parametrize ids can accept None as specific test id. The
   automatically generated id for that argument will be used.
 
-*
+* improved idmaker name selection in case of duplicate ids in
```

Comment:
Please include a "thanks palaviv" and mention #1474. You might want to do the same for the item just above this one. :wink: